reference/api/tasks.mdx

```diff
@@ -232,6 +232,7 @@ Task results are [paginated](/learn/async/paginating_tasks) and can be [filtered
 | **`afterEnqueuedAt`**  | `*` (all tasks)                | [Filter tasks](/learn/async/filtering_tasks) by their `enqueuedAt` field                                                                 |
 | **`afterStartedAt`**   | `*` (all tasks)                | [Filter tasks](/learn/async/filtering_tasks) by their `startedAt` field                                                                  |
 | **`afterFinishedAt`**  | `*` (all tasks)                | [Filter tasks](/learn/async/filtering_tasks) by their `finishedAt` field                                                                 |
+| **`reversed`** | `false` | If `true`, returns results in the reverse order, from oldest to most recent |
```

Comment:
```suggestion
| **`reverse`** | `false` | If `true`, returns results in the reverse order, from oldest to most recent |
```

Oh, sorry, the PRD was wrong (I saw you already noticed).

Don't you think that should be right after the `limit` and `from` since it's related to the way we navigate / paginate instead of doing filtering?